README.md

```diff
@@ -0,0 +1,7 @@
+# aws_connect
+Function that connects user to aws server and performs one of three actions: start server, stop server or check current server state.
+It takes 2 arguments: path to the `.ini` file with all neccessery credentials and action to perform on the server.
+All credentials in the `.ini` file must be in the section `[CREDENTIALS]`.
+<br>
```

Comment:
In markdown if You want to insert a "newline" just add two spaces at the end of a line.